### PR TITLE
Update documentation and disable ORCA for various modules

### PR DESCRIPTION
**Pull request by orhankislal**
- [ ] Add the module name, JIRA# to PR/commit and description.
- [ ] Add tests for the change.

### CODE REVIEW

File: src/ports/postgres/modules/convex/mlp.sql_in [hunk 1]
```diff
@@ -1346,99 +1349,6 @@ JOIN mlp_regress_prediction USING (id);
 (1 row)
 </pre>
 
-<h4>Regression with Grouping and Mini-Batching</h4>
```
Comment (khannaekta) [inline src/ports/postgres/modules/convex/mlp.sql_in:1349]: Why did we remove this?

File: tool/docker_start.sh [hunk 2]
```diff
@@ -83,9 +83,10 @@ docker exec "${CONTAINER_NAME}" bash -c "rm -rf /madlib/build_docker; \
...
-docker exec "${CONTAINER_NAME}" bash -c "/madlib/build_docker/src/bin/madpack -p postgres \
-							 -c postgres/postgres@localhost:5432/postgres \
-							 install" \
+docker exec "${CONTAINER_NAME}" bash -c "PATH=$PATH:/usr/lib/postgresql/10/bin \
+							/madlib/build_docker/src/bin/madpack -p postgres \
+							-c postgres/postgres@localhost:5432/postgres \
+							install" \
 			| tee "${workdir}/build_docker_logs/madlib_install.log"
 
 docker exec "${CONTAINER_NAME}" bash -c "/madlib/build_docker/src/bin/madpack -p postgres \
```
Comment (khannaekta) [inline tool/docker_start.sh:92]: Do we also need to update the PATH here for install-check?

**Review by khannaekta — COMMENTED** [review]
LGTM! Just minor comments.

File: tool/docker_start.sh (hunk 2 above)
Comment (orhankislal) [reply tool/docker_start.sh:92]: No, only the install step needs pg_ctl in the path. Install check or any other madlib function call is fine without it.

File: src/ports/postgres/modules/convex/mlp.sql_in (hunk 1 above)
Comment (orhankislal) [reply src/ports/postgres/modules/convex/mlp.sql_in:1349]: 1- A bit redundant, we already have examples for:
- MLP with grouping
- MLP with minibatching
- minibatching with grouping in the minibatching docs
2- With such a small dataset, it was occasionally failing on my local mac setup.

**Review by khannaekta — APPROVED** [review]

**Pull request merged**